tracer/build/_build/docker/centos7.dockerfile

```diff
@@ -86,31 +86,3 @@ RUN dotnet restore /build
 COPY . /build
 RUN dotnet build /build --no-restore
 WORKDIR /project
-
```

Comment:
Was this stuff supposed to be deleted?

Reply:
Yep: [`cleanup: remove tester stage from centos7 - we don't use it`](https://github.com/DataDog/dd-trace-dotnet/pull/6265/commits/4a02e290085a74bec01828da3663f515859faaa5)